app/routes/events/components/EventAttendeeStatistics.tsx

```diff
@@ -216,19 +223,147 @@ const isEventFromPreviousSemester = (eventStartTime: Dateish): boolean => {
   );
 };
 
+const Analytics = ({ eventId }: { eventId: ID }) => {
+  const [metrics, setMetrics] = useState<{
+    visitors: { title: string; value: number };
+    pageviews: { title: string; value: number };
+    visitDuration: { title: string; value: number };
+  }>({
+    visitors: { title: 'Besøkende', value: 0 },
+    pageviews: { title: 'Sidevisninger', value: 0 },
+    visitDuration: { title: 'Besøkstid', value: 0 },
+  });
+  const [data, setData] = useState<{ date: string; value: number }[]>([]);
+
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    eventId &&
+      dispatch(fetchAnalytics(eventId)).then((res) => {
+        const initialValue = {
+          data: [],
+          visitors: { title: 'Besøkende', value: 0 },
+          pageviews: { title: 'Sidevisninger', value: 0 },
+          visitDuration: { title: 'Besøkstid', value: 0 },
+        };
+
+        const result = res.payload.reduce((acc, item) => {
+          // Update data array
+          acc.data.push({
+            date: item.date,
+            value: item.visitors,
+          });
+
+          // Update metrics
+          acc.visitors.value += item.visitors;
+          acc.pageviews.value += item.pageviews;
+          acc.visitDuration.value += item.visitDuration;
+
+          return acc;
+        }, initialValue);
+
+        setData(result.data);
+        setMetrics({
+          visitors: result.visitors,
+          pageviews: result.pageviews,
+          visitDuration: result.visitDuration,
+        });
+      });
+  }, [eventId, dispatch]);
+
+  return (
+    <>
+      <Flex wrap gap={40} className={styles.metrics}>
+        {Object.values(metrics).map((metric, i) => (
+          <>
+            <Flex column justifyContent="center" key={metric.title}>
+              <span className={styles.metricHeader}>{metric.title}</span>
+              <span className={styles.metricNumber}>
+                {metric.value.toLocaleString('no-NO')}
+              </span>
+            </Flex>
+
+            {i !== Object.values(metrics).length - 1 && (
```

Comment:
Could you write this as `i < Object.keys(metrics).length - 1`? Just a bit easier to understand :smile: